wave_lang/kernel/wave/runtime/runtime.cpp

```diff
@@ -29,12 +29,76 @@ using hipStream_t = void *;
 using hipFunction_t = void *;
 using hipModule_t = void *;
 
+typedef enum hipLaunchAttributeID {
```

Comment:
Should we move the hip specific structs to a separate header file?

Reply:
done